src/modules/CreateDrops/TicketDrop/CreateTicketDropContext/CreateTicketDropContext.tsx

```diff
@@ -104,14 +104,64 @@ export const CreateTicketDropProvider = ({ children }: PropsWithChildren) => {
   });
 
   const getSummaryData = (): SummaryItem[] => {
-    // const { getValues } = methods;
-    // const [eventName] = getValues(['eventName']);
+    const { eventName, totalTickets, additionalGift } = methods.getValues();
 
-    return [];
+    const results: SummaryItem[] = [
+      {
+        type: 'text',
+        name: 'Event name',
+        value: eventName,
+      },
+      {
+        type: 'text',
+        name: 'Number of tickets',
+        value: totalTickets,
+      },
+    ];
+
+    if (additionalGift.type == 'token') {
```

Comment:
let's use `===` instead

Reply:
Ahh sorry, I think it's because of the copilot recommendation haha. Updated here [696872e](https://github.com/keypom/keypom-airfoil/pull/68/commits/696872e4f66e03e8ebb30490cc19138eb67947c1)